centaur/src/main/resources/standardTestCases/three_step.test

```diff
@@ -1,14 +1,20 @@
-name: threeStepCwl
+name: three_step_cwl
 testFormat: workflowsuccess
 workflowType: CWL
 workflowTypeVersion: v1.0
 backends: [Local]
-backendsMode: "only"
 tags: [localdockertest]
 
 files {
   wdl: three_step/three_step.cwl
   inputs: three_step/inputs.json
+  options: three_step/three_step.options
```

Comment:
Does this really fix the problem or just force the JES centaur to run the test on its (should be removed) “Local” backend?

Reply:
Specifically, this forces TES-centaur to run this test on the (should be removed, maybe, see below) "Local" backend.

JES-centaur passes [`-elocaldockertest`](https://github.com/broadinstitute/cromwell/blob/c6ed64617c51c572863b87d324fa8e68fa085b1a/src/bin/travis/resources/centaur.wdl#L34), meaning that _all_ tests tagged as such are skipped. Meanwhile, TES uses a very specific [blacklist](https://github.com/broadinstitute/cromwell/blob/c6ed64617c51c572863b87d324fa8e68fa085b1a/src/bin/travis/testCentaurTes.sh#L71-L78) of incompatible tests, allowing those that can run to specify via a workflow-options file that the test should run on the (should be removed..?) Local.